cmd/vinegar/binary.go

```diff
@@ -106,20 +135,15 @@ func (b *Binary) Main(args ...string) {
 	if firstRun && !sysinfo.CPU.AVX {
 		c := b.Splash.Dialog(DialogNoAVX, true)
 		if !c {
-			log.Fatal("avx is (may be) required to run roblox")
+			return errors.New("avx is (may be) required to run roblox")
```

Comment:
This is an unclear error message, please use the wording "avx may be required to run roblox"

Reply:
Could you please provide a log and error message that indicates that 'AVX may or may not be required to run roblox'? This error isnt related to the user having no AVX, but related to the user acknowledging it may not work.

Change it to simply a dialog message?

Reply:
"avx is missing on this user's system", considering this lives in the log file.